R/module_teal.R

```diff
@@ -136,6 +138,8 @@ srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
       title = "SessionInfo"
     )
 
+    output$lockFile <- teal_lockfile_download()
```

Comment:
`teal_lockfile_download` looks like side-effect download action

how about `teal_lockfile_downloadhandler`? 

BTW? why prefix with the package? Can we do just `lockfile_create()` and `lockfile_downloadhandler()`?

Reply:
I was also thinking about `teal_lockfile_downloadhandler` but though it's too long. If there is two of us thinking about `downloadhandler` then I will use this extended name.

`lockfile_create` is taken by `pak::lockfile_create`. I introduced `teal_` prefix so not to confuse with other packages.

Reply:
teal_lockfile_download -> teal_lockfile_downloadhandler 8e96244488dbbddc5665f642de0057151d8315f4